src/core/Alert/InlineAlert/InlineAlert.baseStyles.ts

```diff
@@ -17,15 +17,15 @@ export const baseStyles = (theme: SuomifiTheme) => css`
       flex: 0;
     }
     &.fi-alert--neutral {
-      border-left: 4px solid #4698c3; /* Later add to theme styling these colors */
+      border-left: 4px solid ${theme.colors.accentSecondary}; /* Later add to theme styling these colors */
```

Comment:
Please remove commets, now when colors are coming fron theme

Reply:
Ah, forgot to remove the comments. Thanks for spotting. They should now be removed :)